frontend/src/static/js/v_summary.js

```diff
@@ -352,22 +366,31 @@ window.vSummary = {
     },
     splitCommitsWeek(user) {
       const { commits } = user;
-      const leng = commits.length;
 
       const res = [];
-      for (let weekId = 0; weekId < (leng - 1) / 7; weekId += 1) {
+
+      const sinceDate = dateRounding(this.filterSinceDate, 1);
+      const untilDate = this.filterUntilDate;
+
+      const sinceMs = (new Date(sinceDate)).getTime();
+      const untilMs = (new Date(untilDate)).getTime();
+
+      const diff = Math.round(Math.abs((untilMs - sinceMs) / DAY_IN_MS));
```

Comment:
here too, do away with the `getTime` conversion

Reply:
actually sinceMs is used elsewhere
`const startOfWeekMs = sinceMs + (weekId * WEEK_IN_MS);`
so I guess still be retained and hence just leave it ?